package.json

```diff
@@ -425,6 +425,16 @@
             "type": "boolean",
             "default": true,
             "markdownDescription": "Load environment files from workspace into all scripts"
+          },
+          "runme.cloud.apiUrl": {
```

Comment:
@degrammer let's please use `.app.` anywhere where it says `.cloud.`. Same goes for the description `Runme Cloud` should be `Runme App`.

Reply:
This is fixed now!